Pieces.sublime-settings

```diff
@@ -1,5 +1,4 @@
 {
-	"host": "",
```

Comment:
Why is host removed? Can the user no longer specify the api path?

Reply:
Yeah it will be automatically detected

Reply:
What if the host is not localhost?

Reply:
I will be removing it for now maybe add it later since sublime only works on the same machine not like vscode it has a remote instance

Reply:
Was there a purpose this feature was available in the first place? if so, what was the purpose?

Reply:
Yeah if someone wants to change the host